packages/shared/src/components/Pill.tsx

```diff
@@ -34,7 +34,7 @@ const Pill = ({
     <div
       className={classNames(
         pillSizeToClassName[size],
-        'rounded-xl',
+        'rounded-12',
```

Comment:
This should be 10px according to design which is `rounded-10` I think I asked dragos before if we had a named variant, but see the numbers are actually our overwrites.

Reply:
I see, in this case, the original value was incorrect being `xl` (12px). Let me fix this.

Reply:
Updated the value to 10 now 🥳